Modules/User/Http/Controllers/ResendVerificationController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+
+namespace Modules\User\Http\Controllers;
+use App\Http\Controllers\Controller;
+use Illuminate\Http\JsonResponse;
+use Modules\User\Services\AuthVerifyService;
+use Illuminate\Http\Request;
+
+
+
+class ResendVerificationController extends Controller{
+    /**
+     * @var AuthVerifyService $authVerifyService
+     */
+    private AuthVerifyService $authVerifyService;
+
+    /**
+     * ResendVerificationController Construct
+     * @param AuthVerifyService $authVerifyService
+     * @return void
+     */
+    public function __construct(AuthVerifyService $authVerifyService) {
+        $this->middleware('auth');
+        $this->middleware('throttle:6,1');
+        $this->authVerifyService = $authVerifyService;
+    }
+
+    /**
+     * @param Request $request
+     * @return JsonResponse
+     */
+    public function __invoke(Request $request) : JsonResponse {
+        $response = $this->authVerifyService->resend($request);
```

Comment:
Me gusto como lo separaste, solo aqui creo que podrias directamente usar $user no? en vez de devolver ese valor de la funcion
$user = $this->info($request->route('id'));

No estas haciendo ninguna modificacion del valor de user, simplemente lo estas enviando y regresando tal cual jaja